### ISSUE TITLE
Alt + Left keyboard shortcut doesn't really work

### ISSUE THREAD

**Issue by kaiyoma** [storybookjs/storybook#12784]
**Describe the bug**
The Alt + Left keyboard shortcut doesn't really work well. It goes to the previous story, but then immediately switches back to the current story. It also conflicts with the browser itself, which uses the same shortcut for navigating backwards.

This was filed previously as issue #8759 but nothing ever happened with it.

**To Reproduce**

1. Press Alt + Left and watch the weirdness.

**Expected behavior**
Storybook should navigate to the previous story.

**System**
```
Environment Info:

  System:
    OS: Windows 10 10.0.19041
    CPU: (4) x64 Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
  Binaries:
    Node: 12.16.1 - C:\Program Files\Node.js\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.13.4 - C:\Program Files\Node.js\npm.CMD
  Browsers:
    Chrome: 86.0.4240.75
    Edge: Spartan (44.19041.423.0), Chromium (86.0.622.38)
  npmPackages:
    @storybook/addon-actions: 6.0.26 => 6.0.26
    @storybook/addon-knobs: 6.0.26 => 6.0.26
    @storybook/addons: 6.0.26 => 6.0.26
    @storybook/core: 6.0.26 => 6.0.26
    @storybook/react: 6.0.26 => 6.0.26
    @storybook/theming: 6.0.26 => 6.0.26
```

**Comment by shilman** [comment]
This appears to be working well for me in 6.1-alpha. Can you verify on your machine/browser?

https://next--storybookjs.netlify.app/official-storybook/?path=/story/addons-a11y-basebutton--invalid-contrast

**Comment by kaiyoma** [comment]
No, it's definitely still an issue. I can repro this problem on your own Storybook deploy:

https://next--storybookjs.netlify.app/official-storybook/

Click around to a few stories, then use Alt + Left and Alt + Right to try to navigate. You should encounter the bad behavior really easily. I can repro by loading the page, opening a story group, clicking to the last story, then trying to use Alt + Left to navigate.

**Comment by kaiyoma** [comment]
Here's a screencast:

![storybook-keyboard-navigation](https://user-images.githubusercontent.com/10719780/96288431-44e16580-0f98-11eb-8322-eb4436ced8e2.gif)

In the video, I first use Alt + Right to navigate forwards, which is fine, but then you can see what happens when I try to use Alt + Left to go back. The app "fights" me and tries to go forward to the current story.

**Comment by shilman** [comment]
@kaiyoma what browser are you using? does it happen in all browsers? in the specific repro you're showing, the accessibility tab is selected (as dictated by those A11y stories. can you repro on other stories in the same storybook where a different tab is selected? I ask because the A11y addon is running a check on the DOM with each render, which could potentially slow down the responsiveness of the app. I'm generally not able to reproduce the problem.

**Comment by kaiyoma** [comment]
Happens all the time in Chrome 86 on Windows 10, but doesn't happen in the same browser in macOS. Maybe an OS-specific issue? The comments in the other (stale) thread mentioned that Storybook's keyboard shortcut interferes with the one already in use by the browser.

I can repro this problem in my project's own Storybook deployment, so I don't think it's related to your a11y stories.

**Comment by shilman** [comment]
Thanks for the clarification @kaiyoma . @ghengeveld @tooppaaa is this something you can look into as part of the sidebar work?

**Comment by ghengeveld** [comment]
Most likely a Windows-only issue.

**Comment by mvz** [comment]
This shortcut interferes with history back in Firefox on Linux as well.

**Comment by valentinpalkovic** [comment]
Thanks for the report. This was on Storybook 6.0.26 and the manager keyboard shortcut module was rewritten when manager-api was consolidated into core in Storybook 9. The conflict with the browser back-shortcut can also be sidestepped by rebinding via `addons.setConfig({ shortcuts: { ... } })`. Closing as stale.
